Sources/graphql-swift/util/networkUtil.swift

```diff
@@ -0,0 +1,46 @@
+import Foundation
+
+enum NetworkRequestError: Error {
+    case NoResponse
+    case NonHttpResponse(URLResponse)
+    case BadResponseCode(Int)
+    case BadData
+    case NonJsonResponse
```

Comment:
🎨   Enum cases should use camel case

```suggestion
    case noResponse
    case nonHttpResponse(URLResponse)
    case badResponseCode(Int)
    case badData
    case nonJsonResponse
```

Reply:
+1